FrontEnd/src/Appointment/AppointmentPage.jsx

```diff
@@ -75,67 +89,109 @@ function AppointmentPage({ userData }) {
     upcomingAppointmentData.length === 0
   ) {
     return (
-      <div className="appointmentPage">
-        <h2>No appointments found.</h2>
+      <div className={`appointmentPage ${theme}`}>
+        <div className="appointments">
```

Comment:
For this, probably better to have a different className like `appointmentsEmptyMessage` so if you want to change the styling for the empty state later on.